cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -63,18 +75,18 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
         initializeConfiguration()
         nodes.forEach { it -> it.installDockerConfig(DEFAULT_SSH_PORT) }
```

Comment:
Seeing as we're here, the `it ->` part of this lambda is not needed because `it` is the name of the default lambda parameter. So
```kotlin
nodes.forEach { it.installDockerConfig(DEFAULT_SSH_PORT) }
```
is technically better.